server/routes/v1/user.ts

```diff
@@ -0,0 +1,42 @@
+import express, { Request, Response } from "express";
+import db from "../../models";
+import EmailService from "../../services/emailService";
+import UserController from "../../controllers/v1/user";
+
+const userRouter = express.Router();
+const eServ = new EmailService();
+const uContr = new UserController(db.User, eServ);
+
+userRouter.get(
+  "/confirmation/c=:token",
+  async (req: Request, res: Response) => {
+    const status: boolean = await uContr.confirmEmail(req.params.token);
```

Comment:
I would check if req.params.token is null. Return a 400. The error message (in the recommended response structure) would be something like: "Missing confirmation token."

Reply:
```suggestion
    const status = await uContr.confirmEmail(req.params.token);
```

I don't think we need to specify the type here.